src/sections/blog-post.liquid

```diff
@@ -0,0 +1,184 @@
+<div class="page-width">
```

Comment:
^^^ Comment block describing:
- This is a required section for the Shopify Theme Store
- What this section does
- What are the required settings
- What are the optional settings

Reply:
I'm still working on the comment block so the text might change.